contracts/Committee.sol

```diff
@@ -425,6 +421,8 @@ contract Committee is ICommittee, Ownable {
 		Participant[] memory _members;
 		uint maxPos;
 
+		// TODO consider modifying computeCommittee to return one list with labels and make a single pass here
```

Comment:
slight nuance - i meant computeComittee will return what it currently does but we pass it's output to some helper function which flattens it's struct to the categorized/labeled list. but perhaps you are thinking of additional simplicifations to computCommittee.. I assumed other places need the split to arrays...